test/autoload.php

```diff
@@ -0,0 +1,11 @@
+<?php // phpcs:disable Generic.Files.LineLength.TooLong
+
+if (PHP_VERSION_ID >= 80100) {
+    require __DIR__ . '/TestAsset/laminas-code/ParameterReflection.php';
+    require __DIR__ . '/TestAsset/laminas-code/MethodReflection.php';
+    require __DIR__ . '/TestAsset/laminas-code/ClassReflection.php';
+
+    class_alias(\Laminas\Code\Reflection\ParameterReflection::class, \Zend\Code\Reflection\ParameterReflection::class);
```

Comment:
Are these needed at all? In theory, we don't use the `Zend\` aliases at all

Reply:
Yes - when running 8.1 against lowest, there's some oddities that happen because the ocramius proxy-manager version is sufficiently low in order to allow usage with 7.3 in this library.

Reply:
do we need usage with 7.3? I checked, and that support was for proxymanager from an ancient past...

Reply:
And, as noted, this is only for testing.

Reply:
We could potentially bump it to 7.4 as the minimum. I've not done that with any of the other packages we've done 8.1 releases for, though. :slightly_frowning_face:

Reply:
Bumped PHP minimum to 7.4. :muscle: